ParkingZone/Areas/Identity/Data/ParkingZoneUser.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace ParkingZone.Areas.Identity.Data;
+
+// Add profile data for application users by adding properties to the ParkingZoneUser class
+public class ParkingZoneUser : IdentityUser
```

Comment:
right now we dont need this model, without ZoneUser, project should work, with solely IdentityUser